src/main/java/org/betonquest/betonquest/api/schedule/Scheduler.java

```diff
@@ -76,8 +77,10 @@ public void addSchedule(final S schedule) {
      * <p><b>
      * When overriding this method, make sure to call {@code super.start()} at some point to update the running flag.
      * </b></p>
+     *
+     * @param now the current time when the scheduler is started
      */
-    public void start() {
+    public void start(final Instant now) {
```

Comment:
If we Introduce a gametime schedule, it is not really based on Instant, so may this need to be an generic?